docs/labels.md

```diff
@@ -9,7 +9,7 @@ Both are specified at the InstanceGroup level.
 
 A nice use for cloudLabels is to specify [AWS cost allocation tags](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html).
 
-A good use for nodeLables is to implement [nodeSelector labels](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#step-two-add-a-nodeselector-field-to-your-pod-configuration). 
```

Comment:
Can we add a link to this doc from the instance group document??

Reply:
done 👍